### PR TITLE
Feature close project

### LINKED ISSUE #39: The Owner can close a project

### PR DESCRIPTION
**Pull request by liannivin**
Status:
Required

Implementation Details:
Add logic to delete a determined project.

close #39

### CODE REVIEW

File: rails-time-track/app/controllers/api/projects_controller.rb [hunk 1]
```diff
@@ -60,6 +60,23 @@ def create
       render_errors(e.message, :not_found)
     end
 
+    def update
+      if @project.update(project_params)
+        render json: @project, status: :ok
+      else
+        render json: {errors: @project.errors}
```
Comment (Linzeur) [inline rails-time-track/app/controllers/api/projects_controller.rb]: What do you think about using the render_errors method which customize the render for error messages. It is in application_controller.rb

File: rails-time-track/config/routes.rb [hunk 2]
```diff
@@ -18,5 +20,6 @@
       post "availableTime", action: "availableTime", on: :collection
     end
     resources :project_members, :only => [:index]
+    resources :pro
```
Comment (Linzeur) [inline rails-time-track/config/routes.rb]: Hi, I think that line is unnecessary, what do you think about removing it?

File: react-time-track/src/views/project.js [hunk 3]
```diff
@@ -34,10 +37,23 @@ const card = {
   width: "auto"
 };
 
-function Project({ project_id }) {
+function Project({ project_id }, props) {
```
Comment (Linzeur) [inline react-time-track/src/views/project.js]: Can you explain me why you are using that props named props?

File: react-time-track/src/components/close-project-modal.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+function CloseProjectModal(props) {
+  return (
+    <Modal isOpen={props.isOpen} onClose={props.onClose}>
+      <div>
+        <h1>Are You Sure?</h1>
+        <p>You are about to delete this project.</p>
+
+        <div>
+          <button onClick={props.onCloseProject} >
```
Comment (Linzeur) [inline react-time-track/src/components/close-project-modal.js]: What do you think about using the button customize which is already the list of components in ui.js?

File: react-time-track/src/views/project.js (hunk 3 above)
Comment (liannivin) [reply react-time-track/src/views/project.js]: I forgot to delete this piece of code. Thanks.

File: rails-time-track/app/controllers/api/projects_controller.rb (hunk 1 above)
Comment (liannivin) [reply rails-time-track/app/controllers/api/projects_controller.rb]: I'll find your method useful, so I used it. Thanks.

**Pull request merged**